test/rbs/ancestor_graph_test.rb

```diff
@@ -83,7 +83,7 @@ module M
           graph.each_ancestor(InstanceNode("::M")).to_set
         )
         assert_equal(
-          Set[InstanceNode("::B"), InstanceNode("::C")],
+          Set[InstanceNode("::B"), InstanceNode("::C"), SingletonNode("::B"), SingletonNode("::C")],
```

Comment:
I think this is a side effect of changing `AncestorBuilder`. But I don't understand if this is a problem or not.